docroot/modules/custom/uiowa_graph/js/csv_graph.js

```diff
@@ -0,0 +1,165 @@
+(($, Drupal, drupalSettings) => {
```

Comment:
I missed our last IE11 conversation - are we still needing to support that? I don't think arrow functions work in IE11: https://caniuse.com/?search=arrow%20functions

Setting up a workspace to transpile this would be overkill though. I just ended up dumbing down my JS in https://github.com/uiowa/uiowa/pull/4013/files#diff-d42454e37e99d62e631edf86d0812c25abdbd610e6c382e13173a0822355deb5.

Reply:
this is a totally valid concern. I think we should use the older version just in case because of IE11 concerns. I can make that change

Reply:
I think re: IE11 we're unofficially in the "as long as it doesn't break, it's fine regardless of how it looks" realm.

Reply:
>as long as it doesn't break, it's fine regardless of how it looks

can we add that to our browser support policy? https://sitenow.uiowa.edu/policies

Reply:
It wasn't that bad to remove them, so no worries. Non-arrow functions are now in place.